src/components/Table/index.tsx

```diff
@@ -427,7 +427,10 @@ class Table<T> extends React.Component<TableProps<T>, TableState<T>> {
 													{item.label || item.field}
 													&nbsp;
 													<FaSort
-														style={{ display: 'inline-block' }}
+														style={{
+															display: 'inline-block',
+															marginBottom: '2px',
```

Comment:
This is because the icon is inherent uneven y padding, so this just evens out that.